Kitodo/src/test/java/org/kitodo/production/services/file/FileServiceIT.java

```diff
@@ -48,12 +61,61 @@ public void testRenameFileWithLockedFile() throws IOException {
             fileService.renameFile(oldUri, "newName.xml");
         } catch (IOException e) {
             URI newUri = URI.create("fileServiceTest/newName.xml");
-            Assert.assertFalse(fileService.fileExist(newUri));
-            Assert.assertTrue(fileService.fileExist(oldUri));
+            assertFalse(fileService.fileExist(newUri));
+            assertTrue(fileService.fileExist(oldUri));
 
         } finally {
             outputStream.close();
         }
 
     }
+
+    @Test
+    public void testCreateDirectories() throws IOException {
+        Path testBaseDirectoryPath = Paths.get(ConfigCore.getParameter(ParameterCore.DIR_XML_CONFIG));
```

Comment:
You added the new test, but you didn't adjust the old ones.
this line needs to be removed here. Please do not give the directory here as the fileManagement is handling it.